module_stats.lua

```diff
@@ -545,6 +551,40 @@ function Module:GetUserStats(guild, userId)
 	return userStats
 end
 
+function Module:GetUserStatsHistory(guild, userId)
+	local guildStatsFolder = self:GetStatsFolder(guild)
+	local files = require("fs").readdirSync(guildStatsFolder)
```

Comment:
why sync?

Reply:
Ah, indeed, easy with this module :smile:

I don't think it's a good idea performance-wise to build dozen of thousands of user histories for a simple `!userinfo`, even if we cache informations, not sure it's worth it.

Reply:
Not doing that means that every call to `!userinfo` will try to read potentially thousands of files for only one user. I think caching it on the first run is a good idea, especially since live stats are kept in memory.

Reply:
True, I will do that then 👌